tests/ui/test_other_user.py

```diff
@@ -0,0 +1,91 @@
+import logging
+import os
+import pytest
+
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.ocs.ui.mcg_ui import ObcUI
+from ocs_ci.ocs import ocp, constants
+from ocs_ci.helpers.helpers import create_unique_resource_name
+from ocs_ci.ocs.ui.views import locators
+from ocs_ci.framework.testlib import ManageTest, ui, bugzilla, polarion_id, tier2
+from time import sleep
+from ocs_ci.utility.utils import ceph_health_check
+
+logger = logging.getLogger(__name__)
+
+
+class ObcUi(ObcUI):
+    def __init__(self, driver):
+        super().__init__(driver)
+        self.sc_loc = locators[self.ocp_version]["obc"]
+
+    def check_obc_option(self, text="Object Bucket Claims"):
+        """check OBC is visible to user after giving admin access"""
+
+        sc_name = create_unique_resource_name("namespace-", "interface")
+        self.do_click(self.sc_loc["Developer_dropdown"])
+        self.do_click(self.sc_loc["select_administrator"], timeout=5)
+        self.do_click(self.sc_loc["create_project"])
+        self.do_send_keys(self.sc_loc["project_name"], sc_name)
+        self.do_click(self.sc_loc["save_project"])
+        self.choose_expanded_mode(mode=True, locator=self.page_nav["Storage"])
+        obc_found = self.wait_until_expected_text_is_found(
+            locator=self.sc_loc["obc_menu_name"], expected_text=text, timeout=10
+        )
+        if not obc_found:
+            logger.info("user is not able to access OBC")
+            self.take_screenshot()
+            return None
+        else:
+            logger.info("user is able to access OBC")
+
+        namespace_obj = OCP(kind=constants.NAMESPACE, resource_name=sc_name)
+        namespace_obj.delete(resource_name=sc_name)
```

Comment:
its giving error that atleast yaml file or resource name required if i put empty bracket.

Reply:
This is weird. Need to debug it.
![image](https://user-images.githubusercontent.com/11279672/187400388-2ba8af40-be1f-4fcb-b448-43f07932a04e.png)

Reply:
What you can do is instead use this [https://github.com/red-hat-storage/ocs-ci/blob/7500cbf39bcd70eb9954807dc860f835efdd1e34/tests/conftest.py#L811](https://github.com/red-hat-storage/ocs-ci/blob/7500cbf39bcd70eb9954807dc860f835efdd1e34/tests/conftest.py#L811) , just send the object as a list.

Reply:
This is maybe because you imported ocp twice. I think you can import only the class and then you'll be sure that the right property from OCP is being used:
from ocs_ci.ocs.ocp import OCP
from ocs_ci.ocs import ocp, constants
I think you should stick with the first import that is importing the class if you don't need functionality outside the class.

Reply:
Please check this thread in the file section. I've added some comments there that are not shown here for some reason.

Reply:
Actually there are multiple delete function which eventually call delete from OCP.py by passing resource name.

Reply:
But why do you need to pass the name when it is already in the object? I've pointed out what is the problem.

Reply:
Done changes as per requested.